src/Core/Migration/Migration1589458026SetDefaultReviewConfig.php

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\Migration\MigrationStep;
+use Shopware\Core\Framework\Uuid\Uuid;
+
+class Migration1589458026SetDefaultReviewConfig extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1589458026;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $connection->insert('system_config', [
+            'id' => Uuid::randomBytes(),
+            'configuration_key' => 'core.listing.showReview',
+            'configuration_value' => '{"_value": true}',
+            'created_at' => (new \DateTime())->format(Defaults::STORAGE_DATE_TIME_FORMAT),
+        ]);
+    }
+
+    public function updateDestructive(Connection $connection): void
+    {
+        // implement update destructive
```

Comment:
could be deleted

Reply:
You mean the comment?
The method itself has to remain. :-)